site/docs/tools/iceberg/spark.md

```diff
@@ -96,7 +99,7 @@ auth_type = "NONE"
     # here we are assuming NONE authorisation
     spark = SparkSession.builder \
             .config("spark.jars.packages",
-                "org.apache.iceberg:iceberg-spark3-runtime-{{ versions.iceberg_spark32 }}:{{ versions.iceberg }},org.projectnessie:nessie-spark-extensions:{{ versions.java }}") \
```

Comment:
Note that this was previously Spark 3.2, I have changed it to Spark 3.1 as it is under the `### Spark 3.1` header (and that fact it is configured to use `NessieSpark32SessionExtensions` rather than `NessieSparkSessionExtensions`.